telegram/_chatinvitelink.py

```diff
@@ -44,7 +44,8 @@ class ChatInviteLink(TelegramObject):
        * Comparing objects of this class now also takes :attr:`creates_join_request` into account.
 
     Args:
-        invite_link (:obj:`str`): The invite link.
+        invite_link (:obj:`str`): The invite link. If the link was created by another chat
+            administrator, then the second part of the link will be replaced with ``'…'``.
```

Comment:
not sure I had to add the 2nd sentence for `invite_link` to args, but it is there in Telegram API

Reply:
Seems like this is recently change in telegram's docs, ig we can change this in argument docs as well :eyes:

Reply:
> we can change this in argument docs as well eyes

Well that's the thing, it is already there in the arguments section. So I thought it might have been deliberately left out of attrs.

Reply:
> we can change this in argument docs as well eyes

Sorry, my deleted comment was nonsense 😆 . I mean that this second sentence was already there in attrs section, so maybe it was deliberately left out of args section because this note may not make a lot of sense in the args.

Reply:
Skipping it in args sounds ok to me

Reply:
Done (removed this phrase from the args again, the way it used to be)